Formula/streamsets.rb

```diff
@@ -0,0 +1,50 @@
+class Streamsets < Formula
+  desc "ETL tool with a graphical IDE for building dataflow pipelines"
+  homepage "https://streamsets.com/"
+  url "https://archives.streamsets.com/datacollector/1.5.1.0/tarball/streamsets-datacollector-all-1.5.1.0.tgz"
+  sha256 "d848ffaec4cbadce0e7aa031fb30d1e7e13c84be9115ecd6c69643ff02400fc8"
+
+  bottle :unneeded
+
+  depends_on :java => "1.7+"
+  depends_on "md5sha1sum"
+
+  def install
+    # rename old SDC_CONF if it exists
+    if File.exist?("#{var}/lib/sdc")
+      mv "#{etc}/sdc", "#{etc}/sdc-old", :force => true
+    else
+      (var/"log/sdc").mkpath
+      (var/"lib/sdc-resources").mkpath
+      (var/"lib/sdc").mkpath
+      (etc/"sdc").mkpath
+    end
+
+    libexec.install Dir["*"]
+
+    ENV["STREAMSETS_HOME"] = libexec
+
+    bin.install libexec/"bin/streamsets" => "streamsets"
+    bin.env_script_all_files libexec/"bin/", :STREAMSETS_HOME => libexec
+
+    # edit sdc-env.sh so that data is stored in ${HOMEBREW_PREFIX}/var and ${HOMEBREW_PREFIX}/etc, so that data will be saved after brew upgrade
```

Comment:
Probably alright without this comment, we do this across Homebrew enough that it's likely pretty obvious what we're doing.